feature/google/src/test/java/com/revenuecat/purchases/google/StoreProductTest.kt

```diff
@@ -1,94 +1,60 @@
 package com.revenuecat.purchases.google
 
 import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.android.billingclient.api.BillingClient
+import com.revenuecat.purchases.utils.mockProductDetails
+import com.revenuecat.purchases.utils.mockSubscriptionOfferDetails
+import org.assertj.core.api.AssertionsForInterfaceTypes.assertThat
+import org.junit.Test
 import org.junit.runner.RunWith
 
-//    TODOBC5: After using mocks for ProductDetails, these tests don't make much sense. Leaving for now.
```

Comment:
I think it's still worth it to keep these tests. The comment above is outdated, but I think having a test for the `equals` of the `GoogleStoreProduct` among some others can be useful.